portfolio/src/main/webapp/script.js

```diff
@@ -46,23 +66,32 @@ function getComments() {
         addComment(comment)
       );
     });
+    Object.keys(data).length > 0 ? toggleDeleteAllButton(true) : toggleDeleteAllButton(false);
```

Comment:
```toggleDeleteAllButton(Object.keys(data).length > 0)```?

Reply:
I made it so the "Delete all comments" button only appears when comments exist- basically when the comment count > 0.